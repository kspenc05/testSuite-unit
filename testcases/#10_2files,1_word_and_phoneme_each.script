#!/bin/sh
./cmulookup cmudict-0.7b input/1word.txt input/1word_again.txt