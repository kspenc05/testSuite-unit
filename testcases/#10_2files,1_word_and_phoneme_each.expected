K AW1 CH
=====
HH OW1 M L AH0 S
=====
DONE! (2 words)
